### ISSUE TITLE
event kind bit flags

### ISSUE THREAD

**Issue by mikedilger** [mikedilger/nostr-next#20]
Microblogs may have different content types, plaintext, markdown, etc.  Fiatjaf has argued for using different kinds for this.  But then we need a whole class of event kinds which are treated as feed related, displayable, and use the inbox/outbox model.

It makes sense to me to have event kind bit flags, where if the microblog flag is up, that means it is feed related, displayable and uses the inbox/outbox model.  Then other flags could mean other things.

**Comment by melvincarvalho** [comment]
Yes, I think it's needed as nostr grows.  In linked data kinds are called Types.  Or you can think of Classes in OO.  Typically there is a one to one mapping.  But as use cases grow you will often see multiple types on an entity.  That is impossible with kinds.  So a bitflag might be a good compromise way of doing this.
